metofficedatahub/app.py

```diff
@@ -0,0 +1,53 @@
+""" Application that pulls data from the Metoffice API and saves to a zarr file"""
+import logging
+
+import click
+
+from metofficedatahub.multiple_files import MetOfficeDataHub, save_to_zarr
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.option(
+    "--api-key",
+    default=None,
+    envvar="API_KEY",
+    help="The API key for MetOffice Weather DataHub AMD",
```

Comment:
```suggestion
    help="The API key for MetOffice Weather DataHub",
```

Is there a different API key for AMD and non-AMD?
If not I think would be good to remove the `AMD` here and below